src/devices/Seesaw/README.md

```diff
@@ -0,0 +1,46 @@
+﻿# Adafruit Seesaw
+
+## Summary
+Adafruit Seesaw is a near-universal converter framework which allows you to add add and extend hardware support to any I2C-capable microcontroller or microcomputer. Instead of getting separate I2C GPIO expanders, ADCs, PWM drivers, etc, seesaw can be configured to give a wide range of capabilities.
+
+This binding procvides an Api which is close to the one provided by Adafruit themselves but also implements IGpioController so that available Gpio pins can be used in place of any 'on board' ones but using the standard IoT API.
```

Comment:
```suggestion
This binding provides an Api which is close to the one provided by Adafruit themselves but also implements IGpioController so that available Gpio pins can be used in place of any 'on board' ones but using the standard IoT API.
```

Reply:
Done in the latest PR update.